test/unit/utils/personutils.test.js

```diff
@@ -155,4 +156,205 @@ describe('personutils', function() {
+    });
+
+    it('should return error message when diagnosisDate is in the future', function() {
+      var formValues = {
+        fullName: 'Joe Bloggs',
+        birthday: '01/05/1984',
+        diagnosisDate: '01/01/2016',
+        about: null
+      };
+      var error = personUtils.validateFormValues(formValues, true, FORM_DATE_FORMAT, Date.UTC(2015, 4, 18));
+
+      expect(error.diagnosisDate).to.equal('Diagnosis date cannot be in the future!');
+    });
+
+    it('should return error message when diagnosisDate is before birthday', function() {
+      var formValues = {
+        fullName: 'Joe Bloggs',
+        birthday: '01/05/1984',
+        diagnosisDate: '01/01/1983',
+        about: null
+      };
+      var error = personUtils.validateFormValues(formValues, true, FORM_DATE_FORMAT, Date.UTC(2015, 4, 18));
+
+      expect(error.diagnosisDate).to.equal('Diagnosis cannot be before date of birth!');
+    });
+
+    it('should return no error message when diagnosisDate and birthday and about is valid', function() {
+      var formValues = {
+        fullName: 'Joe Bloggs',
+        birthday: '01/01/1984',
+        diagnosisDate: '01/05/1984',
+        about: 'This is a valid length about section'
+      };
+      var error = personUtils.validateFormValues(formValues, true, FORM_DATE_FORMAT);
+
+      expect(Object.keys(error).length).to.equal(0);
+    });
+
+    it('should return error message when about is over max length', function() {
+      var formValues = {
+        fullName: 'Joe Bloggs',
+        birthday: '01/01/1984',
+        diagnosisDate: '01/05/1984',
+        about: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ' +
+        'Enim in consectetur ultricies netus torquent nisi gravida pulvinar' +
+        ' - curae congue tellus sodales nec proin?Risus in nostra montes rhoncus' +
+        ' vestibulum tempus per ut: curae maecenas nibh arcu eget. Dolby'
+      };
+      var error = personUtils.validateFormValues(formValues, true, FORM_DATE_FORMAT);
+
+      expect(error.about).to.equal('Please keep "about" text under 256 characters');
+    }); 
+
+    it('should return no error message when diagnosisDate and birthday and about is at max length', function() {
+      var formValues = {
+        fullName: 'Joe Bloggs',
+        birthday: '01/01/1984',
+        diagnosisDate: '01/05/1984',
+        about: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ' + 
+        'Enim in consectetur ultricies netus torquent nisi gravida pulvinar' + 
+        ' - curae congue tellus sodales nec proin?Risus in nostra montes rhoncus' + 
+        ' vestibulum tempus per ut: curae maecenas nibh arcu eget. Dolb'
+      };
+      var error = personUtils.validateFormValues(formValues, true, FORM_DATE_FORMAT);
+
+      expect(Object.keys(error).length).to.equal(0);
+    });
+
+    it('should return multiple error messages when multiple validation problems', function() {
```

Comment:
I added one test here - partly I was "thinking out loud" as I was seeing how you did the refactor, but also I think it's a helpful test for additional documentation of how the validation works.